src/style.css

```diff
@@ -1 +1,13 @@
-/* styles go here */
+.form__field {
```

Comment:
you should add this class to all of your fields. 
And you can just use 
```
.form__field:not(:last-child) {
  margin-bottom: 20px
}
```

the same is for fieldset